README.md

```diff
@@ -867,6 +867,7 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 ### Configuration
 
 * [mehcode/config-rs](https://github.com/mehcode/config-rs) [[config](https://crates.io/crates/config)] — Layered configuration system for Rust applications (with strong support for 12-factor applications). [<img src="https://api.travis-ci.org/mehcode/config-rs.svg?branch=master">](https://travis-ci.org/mehcode/config-rs)
+* [theimpossibleastronaut/configster](https://github.com/theimpossibleastronaut/configster) [[configster](https://crates.io/crates/configster)] — Rust library for parsing configuration files [<img src="https://travis-ci.com/theimpossibleastronaut/configster.svg?branch=trunk">](https://travis-ci.com/theimpossibleastronaut/configster)
```

Comment:
The build badge 301's to https://api.travis-ci.com/theimpossibleastronaut/configster.svg?branch=trunk so could you please change the URL for it?